emissionsapi/web.py

```diff
@@ -70,6 +73,15 @@ def get_data(session, country=None, geoframe=None, begin=None, end=None):
         if country not in country_bounding_boxes:
             return 'Unknown country code.', 400
         rectangle = bounding_box_to_wkt(*country_bounding_boxes[country][1])
+    # parse parameter wkt
+    elif polygon is not None:
+        if len(polygon) % 2 != 0:
```

Comment:
You check this in `polygon_to_wkt(…)` anyway.

Reply:
Yes, but here I can create a dedicated response message. So the question is, if this will be a common mistake and a dedicated message is appropriate?

Reply:
See the comment below which would make this return the actual error message

Reply:
Very nice!